pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cookiecutter-poetry"
-version = "0.1.0"
+version = "0.1.1"
```

```suggestion
version = "0.1.0"
```

This can be kept unchanged.

---

Did this, thanks.